pwiz_tools/Skyline/Skyline.cs

```diff
@@ -2175,9 +2176,17 @@ private void ShowPeptideSettingsUI(IWin32Window parent, PeptideSettingsUI.TABS?
         private void HandleStandardsChanged(ICollection<Target> oldStandard)
         {
             var calc = RCalcIrt.Calculator(Document);
-            if (calc == null || !File.Exists(calc.DatabasePath))
+            if (calc == null)
                 return;
-            calc = calc.Initialize(null) as RCalcIrt;
+            try
+            {
+                calc = calc.Initialize(null) as RCalcIrt;
+            }
+            catch (Exception)

```

Comment:
Should we really be returning silently? I would much prefer that the user see a message about what is going on that may allow them to fix the problem, (e.g. file missing, access restriction, database corruption, etc.)

We don't want the unhandled exception form, but we do want to show them some information. Code like this doesn't necessarily reduce support. It may cause more confusion, and because the exception may be system dependent, it may be very challenging for us to debug by asking for file to be sent to us.